### DISCUSSION TITLE
Component Class not found on template

### DISCUSSION THREAD

**Discussion by Jashandeep-1229** [netflie/whatsapp-cloud-api#12] [Q&A] [answered]
Please help me to solve this issue (Component Not Found) in laravel

**Comment by aalbarca** [comment, marked as answer]
You have to import the Component class `use Netflie\WhatsAppCloudApi\Message\Template\Component`

**Reply by Jashandeep-1229** [reply]
Thank you! And One More How can I send the image through the component body?
